validator/client/beacon-api/duties.go

```diff
@@ -209,7 +214,16 @@ func (c *beaconApiValidatorClient) dutiesForEpoch(
 		}
 	}
 
-	return duties, nil
+	dutiesContainer.CurrentEpochDuties = duties
+	dutiesContainer.CurrDependentRoot, err = hexutil.Decode(proposerDutiesContainer.DependentRoot)
+	if err != nil {
+		return errors.Wrap(err, "failed to decode current dependent root")
+	}
+	dutiesContainer.PrevDependentRoot, err = hexutil.Decode(attesterDutiesContainer.DependentRoot)
```

Comment:
oh... i didn't realize the dependent root on attester duties was the previous dependent root...